kafka-client/src/main/java/dev/responsive/kafka/internal/db/mongo/MongoKVTable.java

```diff
@@ -210,4 +207,11 @@ public long approximateNumEntries(final int kafkaPartition) {
     return 0;
   }
 
+  private ObjectId idForKafkaPartition(final int kafkaPartition) {
```

Comment:
I'm pretty sure you can change the `_id` field type if you want, in this case we should just make it an `int` instead of `ObjectId`.

Reply:
https://github.com/responsivedev/responsive-pub/pull/210